src/include/daos_srv/bio.h

```diff
@@ -686,4 +688,14 @@ bool bio_need_nvme_poll(struct bio_xs_context *xs);
  */
 int bio_replace_dev(struct bio_xs_context *xs, uuid_t old_dev_id,
 		    uuid_t new_dev_id);
+
+/*
+ * Identify a VMD device.
+ *
+ * \param xs            [IN]    xstream context
+ * \param traddr        [IN]    Transport ID of the VMD device
+ *
+ * \return                      Zero on success, negative value on error
+ */
+int bio_identify_dev(struct bio_xs_context *xs, char *traddr);
```

Comment:
It's better to introduce a more generic interface, like bio_set_led_state(), so that it can be reused for setting any LED state.

Reply:
done